### PR TITLE
feat: add some documentation about rock&gazebo

### PR DESCRIPTION
**Pull request by doudou**

### CODE REVIEW

**Comment by doudou** [comment]
@afonsofonbraga could you have a look ?

File: source/simulation/models_links_and_joints.html.md [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+
+~~~
+<model name="wet_paint">
+    <link name="camera" />
+</model>
+~~~
+
+the device that represents the model will be named `wet_paint` and the one for
+the link `wet_paint_camera_link`, respectively accessible via `Base.wet_paint_dev` and
+`Base.wet_paint_camera_link_dev`. See [our tutorial](../basics/devices.html) for
```
Comment (rodrigoleonello) [inline source/simulation/models_links_and_joints.html.md:52]: This link is not working...

File: source/simulation/models_links_and_joints.html.md [hunk 2]
```diff
@@ -0,0 +1,257 @@
...
+                # prefix_device_with_name is a backward compatibility thing.
+                # Just pass true.
+                use_gazebo_model "model://ModelName", prefix_device_with_name: true
+            end
+        end
+    end
+end
+~~~
+
+This stanza will define a number of devices for [links](#links), [sensors](#sensors) and [plugins](plugins.html)
```
Comment (rodrigoleonello) [inline source/simulation/models_links_and_joints.html.md:38]: The link is not working...should be `plugins.html.md`?

Comment (doudou) [reply source/simulation/models_links_and_joints.html.md:38]: It ... should not. If you want to test the site pre-rendered, you need to run the middleman vscode task of rock-and-syskit and open your browser there. vscode is not enough.

**Review by caioaamaral — APPROVED** [review]

**Pull request merged**